mitosheet/mitosheet/tests/updates/test_existing_import.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# Copyright (c) Saga Inc.
+# Distributed under the terms of the GPL License.
+"""
+Contains tests for existing import update events.
+"""
+import pandas as pd
+import os
+from mitosheet.tests.test_utils import create_mito_wrapper, create_mito_wrapper_dfs
+from mitosheet.tests.decorators import pandas_post_1_only, python_post_3_6_only
+
+TEST_EXCEL_FILE = 'excel_file.xlsx'
+TEST_CSV_FILE = 'csv_file.csv'
+TEST_CSV_FILE_TWO = 'csv_file_two.csv'
+
+
+@pandas_post_1_only
+@python_post_3_6_only
```

Comment:
Why these?

Reply:
Cause we're using the excel import which requires this I believe. I took it becuase the Excel import tests were marked with it.